File: python/Algorithm5.py
# Algorithm 5: Check whether a number is prime or not

n, i, flag = 0, 2, 1
n = int(input("Enter a number: "))

while i<= n//2:
    if n%i==0:
        flag=0
        break
    i+=1

if flag==0:
    print(n, " is not prime")
else:
    print(n, " is prime")

File: javascript/Algorithm5.js
// Algorithm 5: Check whether a number is prime or not

const readline = require("readline");

let n, i, flag = 1;
i=2;

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter a number: ", function(input) {
    n = parseInt(input);

    while (i<=n/2) {
        if(n%2==0) {
            flag=0;
            break;
        }
        i++;
    }

    if(flag==0) {
        console.log(n + " is not prime");
    } else {
        console.log(n + " is prime");
    }

    rl.close();
});